gulpfile.js

```diff
@@ -440,6 +440,7 @@ function buildExtensionJs(path, name, version, options) {
     // See https://github.com/ampproject/amphtml/issues/3977
     wrapper: options.noWrapper ? '' : ('(self.AMP=self.AMP||[])' +
         '.push({n:"' + name + '",' + priority +
+        'v:"' + internalRuntimeVersion + '",' +
```

Comment:
I have a comment at the use site for this. We care about the version of the code here, not about the version of the config.